sktime/transformers/series_as_features/tests/test_derivative_transformer.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import pandas as pd
+import pytest
+import math
+
+from sktime.exceptions import NotFittedError
+from sktime.transformers.series_as_features.derivative \
+    import DerivativeTransformer
+from sktime.utils.data_container import tabularize
+from sktime.utils._testing import generate_df_from_array
+
+            
+# Check that NotFittedError is thrown if someone attempts to
+# transform before calling fit
+def test_early_trans_fail():
```

Comment:
that's a good test, but we test all estimators that inherit from out base classes for basic things like this automatically, so no need to do it here again :)